lib/cep/sources/postmon.ex

```diff
@@ -16,7 +16,9 @@ defmodule Cep.Sources.Postmon do
 
         {:ok, address}
 
-      {:ok, %HTTPoison.Response{status_code: 404}} ->
+      # For some reason Poison is returns 503 and it
+      # returns { :ok, %{ status_code: 503 } }
```

Comment:
How can you reproduce this? I tried with a normal CEP and one that returns not found and it works as expected.

Reply:
try with: `01415-111`

Reply:
![image](https://user-images.githubusercontent.com/503670/55368773-12f9d280-54ca-11e9-8f83-93c54735fec8.png)

Reply:
Cool! I will have a look at their source code and see if I can find the reason for the "good response" with a 503. Maybe I could fix it or at least fill an issue and mention it here.

I suspect this might break the tests though. Can you have a look at the merge conflict so travis can build the PR?

Reply:
I just rebased... now I think it will work :)